iOS/Swift/IronSourceSwiftDemoApp/DemoViewController.swift

```diff
@@ -206,29 +230,22 @@ class DemoViewController: UIViewController, DemoViewControllerDelegate {
                 buttonToModify = self.showInterstitialButton
             case .loadBannerButtonIdentifier:
                 buttonToModify = self.loadBannerButton
-            case .destroyBannerButtonIdentifier:
-                buttonToModify = self.destroyBannerButton
             }
             
             buttonToModify?.isEnabled = enable
         }
     }
     
-    func setAndBindBannerView(_ bannerView: ISBannerView!) {
+    func didLoadBannerAd() {
         DispatchQueue.main.async {
-            if (self.bannerView != nil) {
-                self.bannerView.removeFromSuperview()
-            }
-            
-            self.bannerView = bannerView
-            self.bannerView.translatesAutoresizingMaskIntoConstraints = false
-            self.view.addSubview(bannerView)
-            
-            let centerX = self.bannerView.centerXAnchor.constraint(equalTo: self.view.centerXAnchor)
-            let bottom = self.bannerView.bottomAnchor.constraint(equalTo: self.view.safeAreaLayoutGuide.bottomAnchor)
-            let width = self.bannerView.widthAnchor.constraint(equalToConstant: bannerView.frame.size.width)
-            let height = self.bannerView.heightAnchor.constraint(equalToConstant: bannerView.frame.size.height)
+            self.bannerAdView.translatesAutoresizingMaskIntoConstraints = false
+            self.view.addSubview(self.bannerAdView)
+            let centerX = self.bannerAdView.centerXAnchor.constraint(equalTo: self.view.centerXAnchor)
+            let bottom = self.bannerAdView.bottomAnchor.constraint(equalTo: self.view.safeAreaLayoutGuide.bottomAnchor)
+            let width = self.bannerAdView.widthAnchor.constraint(equalToConstant: CGFloat(self.bannerSize.width))
+            let height = self.bannerAdView.heightAnchor.constraint(equalToConstant: CGFloat(self.bannerSize.height))
```

Comment:
there is no reason to make these constraints every time because it's the same object